fuzz/fuzz_targets/full_stack_target.rs

```diff
@@ -279,13 +279,14 @@ impl KeysInterface for KeyProvider {
 	}
 
 	fn get_session_key(&self) -> SecretKey {
-		let ctr = self.counter.fetch_add(1, Ordering::Relaxed);
+		let ctr = self.counter.fetch_add(1, Ordering::Relaxed) as u8;
 		SecretKey::from_slice(&[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 13, ctr]).unwrap()
 	}
 
 	fn get_channel_id(&self) -> [u8; 32] {
 		let ctr = self.counter.fetch_add(1, Ordering::Relaxed);
-		[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 14, ctr]
+		[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+		(ctr >> 8*7) as u8, (ctr >> 8*6) as u8, (ctr >> 8*5) as u8, (ctr >> 8*4) as u8, (ctr >> 8*3) as u8, (ctr >> 8*2) as u8, (ctr >> 8*1) as u8, 14, (ctr >> 8*0) as u8]
```

Comment:
Fuzzing got a channel collision on 256-bit id for two subsequent channels ?!

Reply:
Sure, just gotta call one of the functions above exactly 256 times.